src/interfaces/database/ICiraConfigDb.ts

```diff
@@ -4,7 +4,7 @@
  * Author : Ramu Bachala
  **********************************************************************/
 import { CIRAConfig } from '../../RCS.Config'
-import { IDB } from './IDb'
+import { ITable } from './ITable'
 
-export interface ICiraConfigDb extends IDB<CIRAConfig> {
+export interface ICiraConfigTable extends ITable<CIRAConfig> {
 }
```

Comment:
Is it to follow the hierarchy and for future use? Otherwise we can directly extends ITable<CIRAConfig> right?

Reply:
Exactly -- going for consistency here and planning for optimizations we can make in the future for more specific calls. But yes, can technically extend ITable directly.